src/bpfbytecode.h

```diff
@@ -58,6 +61,7 @@ class BpfBytecode {
   std::unique_ptr<struct bpf_object, bpf_object_deleter> bpf_object_;
 
   std::map<std::string, BpfMap> maps_;
+  std::map<int, BpfMap *> maps_by_id_;
```

Comment:
If the map is non-nullable here, maybe better to store a reference?

Reply:
I don't think it's possible to store a reference in a `std::map`. We could use `std::reference_wrapper` but that feels like adding an unnecessary complexity to me.

Reply:
Ah true, yeah. Ptr is fine